src/Statement/Filter/PropertySetStatementFilter.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Wikibase\DataModel\Services\Statement\Filter;
+
+use Wikibase\DataModel\Statement\Statement;
+use Wikibase\DataModel\Statement\StatementFilter;
+
+/**
+ * @since 3.2
+ *
+ * @licence GNU GPL v2+
+ * @author Thiemo Mättig
+ */
+class PropertySetStatementFilter implements StatementFilter {
+
+	/**
+	 * @var string[]
+	 */
+	private $propertyIds;
+
+	/**
+	 * @param string[]|string $propertyIds One or more property id serializations.
+	 */
+	public function __construct( $propertyIds ) {
+		$this->propertyIds = (array)$propertyIds;
+	}
+
+	/**
+	 * @see StatementFilter::statementMatches
+	 *
+	 * @param Statement $statement
+	 *
+	 * @return bool
+	 */
+	public function statementMatches( Statement $statement ) {
+		$id = $statement->getPropertyId()->getSerialization();
+		return in_array( $id, $this->propertyIds );
+	}
+
+}
```

Comment:
Without the most basic implementation(s) it's almost pointless to put these interfaces here, in a component other users can and should use. If you plan to remove everything that's not going to be used in Wikibase right away, with this being the only reason, then please abandon this patch completely and implement what's needed in WikibaseView. You can most probably ditch all the interfaces then and use the implementations directly. Or even go with @brightbyte's https://github.com/wmde/WikibaseDataModelServices/pull/91#discussion_r46144648.